spec/guardtime_driver_spec.rb

```diff
@@ -1,27 +1,43 @@
 require "spec_helper"
 require "date"
 
-RSpec.shared_examples "Guardtime" do |gt_driver|
-  username = ENV['GUARDTIME_user']
-  password = ENV['GUARDTIME_password']
-  now = DateTime.now
+username = ENV['GUARDTIME_user']
```

Comment:
This VCR config can be moveed into a separate file, perhaps `spec/support/vcr.rb`, and then features that need it could be tagged using RSpec metadata. See https://relishapp.com/vcr/vcr/v/3-0-3/docs/test-frameworks/usage-with-rspec-metadata for details.

Reply:
Sorry I see you're using the metadata approach already - this config just looked a bit unusual so I didn't think you were.

Reply:
I'd still suggest moving into a shared support file.